tests/unit/s2n_extensions_server_key_share_select_test.c

```diff
@@ -35,7 +36,10 @@ int main(int argc, char **argv)
          */
         EXPECT_NOT_NULL(server_conn = s2n_connection_new(S2N_SERVER));
         EXPECT_NULL(server_conn->secure.server_ecc_evp_params.negotiated_curve);
-        for (int i = 0; i < S2N_ECC_EVP_SUPPORTED_CURVES_COUNT; i++) {
+        notnull_check(server_conn->config);
```

Comment:
Same issue pointed out on another test-- use EXPECT_NOT_NULL, not notnull_check in tests